src/components/nodes/subProcess/SubProcessFormSelect.vue

```diff
@@ -2,30 +2,38 @@
   <div>
     <form-multi-select
       label="Process"
+      name="Process"
       helper="Select which Process this element calls"
       v-model="selectedProcess"
+      :showLabels="false"
+      :allow-empty="false"
       :disabled="processList.length === 0"
       :options="processList"
       optionContent="name"
       class="p-0 mb-2"
+      :validation="'required'"
```

Comment:
Since you're passing in a string, you don't need to explicitly bind a string value.
```suggestion
      validation="required"
```